src/assets/translations/en/main.json

```diff
@@ -1276,6 +1277,17 @@
         "success": {
           "title": "Success",
           "description": "You have backed up your wallet. Remember to store your Secret Recovery Phrase somewhere safe."
+        },
+        "error": {
+          "title": "Incorrect",
+          "description": "One or more of the selected words were incorrect.",
+          "reviewPhrase": "Review Secret Recovery Phrase"
+        },
+        "skip": {
+          "title": "Are you sure you want to skip?",
```

Comment:
Product: Not sure about this copy, feels odd from a user perspective. Should either be fully verbose (`Are you sure you want to skip backup?`) or a shorthand (`Are you sure?`) but this feels like an odd mix.